wandb/sdk/internal/tb_watcher.py

```diff
@@ -419,21 +418,19 @@ def _handle_event(self, event: "ProtoEvent", history: "TBHistory" = None) -> Non
         )
 
     def _save_row(self, row: "HistoryDict") -> None:
-        chart_keys = []
-        for key, item in row.items():
-            if isinstance(item, CustomChart):
-                panel_config = custom_chart_panel_config(item, key, key + "_table")
-                config = {"panel_type": "Vega2", "panel_config": panel_config}
-                chart_keys.append(key)
-                self._tbwatcher._interface.publish_config(
-                    val=config, key=("_wandb", "visualize", key)
+        chart_keys = set()
+        for k in row:
+            if isinstance(row[k], CustomChart):
+                chart_keys.add(k)
+                key = row[k].get_config_key(k)
```

Comment:
won't this log the visualization at key `_wandb.visualize.key`?

Reply:
yeah that's what we had before didn't we?

Reply:
Oh, oops the k we add to chart keys is NOT modified with the prefixes